test/testinput/bufrtest_bufrdescription.yaml

```diff
@@ -0,0 +1,77 @@
+# (C) Copyright 2020 NOAA/NWS/NCEP/EMC
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+inputpath: "/Users/rmclaren/Work/sample-bufr-data/gdas/gdas.20200704/12"
+outputpath: "/Users/rmclaren/Temp/"
+
+
+bufr:
+  filename: "gdas.t12z.1bmhs.tm00.bufr_d"
+
+  mnemonicSets:
+    - mnemonics: [SAID, FOVN, YEAR, MNTH, DAYS, HOUR, MINU, SECO, CLAT, CLON, CLATH, CLONH, HOLS]
+    - mnemonics: [SAZA, SOZA, BEARAZ, SOLAZI]
+    - mnemonics: [TMBR]
+      channels : 1-15
+
+  exports:
+    timestamp:
+      datetime:
+        year: YEAR
+        month: MNTH
+        day: DAYS
+        hour: HOUR
+        minute: MINU
+        second: SECO
+        isUTC: true
+
+    longitude:
+      mnemonic: CLON
+      transforms:
+        - offset: -180
+        - scale: 1.25
+
+    latitude:
+      mnemonic: CLAT
+    radiance:
+      mnemonic: TMBR
+
+ioda:
+  filename: "ioda_encoder_result.nc"
```

Comment:
For now this is fine, but in the future, we should be able to provide different backends e.g. netcdf, odb, in-memory, etc, rather than just a filename.

Reply:
agreed